notebooks/Text_Extensions_for_Pandas_Overview.ipynb

```diff
@@ -0,0 +1,1819 @@
+{
```

Comment:
<p>Add a small blurb about the crude tokenizer in our case is just splitting the paragraph into individual words. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>